README.md

```diff
@@ -1,11 +1,39 @@
-# gdp-submodules
-GDP setup using git submodules
+About GDP: GENIVI Development Platform
+================================
```

Comment:
While I'm commenting, how do you decide how wide to make the rule here? I believe 3 is the minimum, and perfectionists tend to have as many as the above line.

It won't stop merging, but I'd like to know your thoughts on it.